rest/model/user.go

```diff
@@ -1,8 +1,11 @@
 package model
 
 import (
+	"reflect"
+
 	"github.com/evergreen-ci/evergreen/model/user"
 	"github.com/pkg/errors"
+	"gopkg.in/mgo.v2/bson"
```

Comment:
we should remove this dependency by whatever means necessary.

Reply:
with fire